### PR TITLE
Revert "Upgrade OpenTelemetry (#3675)"

### PR DESCRIPTION
**Pull request by macfarla**
This reverts commit 78717ade1d45902c13e77e84bc201338e159c3c4.

Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

Issues with "Failed to connect" errors 

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by shemnon — APPROVED** [review]

File: acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/OpenTelemetryAcceptanceTest.java [hunk 1]
```diff
@@ -139,26 +131,16 @@ public void setUp() throws Exception {
     MetricsConfiguration configuration =
         MetricsConfiguration.builder()
             .protocol(MetricsProtocol.OPENTELEMETRY)
-            .pushEnabled(true)
+            .enabled(true)
             .port(0)
             .hostsAllowlist(singletonList("*"))
             .build();
-    Map<String, String> env = new HashMap<>();
```
[the remark this answers is not in the document]
Comment (macfarla) [reply acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/OpenTelemetryAcceptanceTest.java:146]: this is in the acceptance test though?

**Comment by atoulme** [comment]
What’s going on?

**Comment by macfarla** [comment]
> What’s going on?

Sorry - the context is in Discord threads 

I'm getting a stack of these messages when running unit tests locally but others are seeing this when running latest besu -
June 28, 2022 12:54:14 PM io.opentelemetry.sdk.internal.ThrottlingLogger doLog
SEVERE: Failed to export spans. The request could not be executed. Full error message: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:4317

File: acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/OpenTelemetryAcceptanceTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (atoulme) [reply acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/OpenTelemetryAcceptanceTest.java:146]: No that’s not it, sorry.

**Comment by atoulme** [comment]
Got it. Will look.

**Pull request merged**